llvm/lib/SYCL/InSPIRation.cpp

```diff
@@ -348,78 +348,6 @@ struct InSPIRation : public ModulePass {
     }
   }
 
-  /// This currently looks through the arguments passed to the SSDM intrinsic
```

Comment:
So you are removing the support for the old style SSDM?
I guess it is no longer used anyway @yu810226 @aisoard 
By the way, I am expecting the same level of comments elsewhere in the code... :-)